api/src/main/java/io/grpc/CallOptions.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 The gRPC Authors
+ * Copyright 2015, 2022 The gRPC Authors
```

Comment:
I don't think we add or update the year in an existing copyright notice. Have you seen this elsewhere in the grpc code?